### PR TITLE
Change variable name so that it doesnt conflict with other make files

### PR DESCRIPTION
**Pull request by ed-sparkes**
## Link to JIRA Ticket

- DI-64

## Description

The original PR for DI-64 overwrote the PYTHON_VERSION variable in project.mk to major.minor however other make files specifically the python-install one relied on it being major.minor.patch

### Noteworthy Changes

- Renamed the variable in project.mk

## Type of change

Delete not appropriate

- Bug fix (non-breaking change which fixes an issue)
- 
## Testing

Please tick the testing that has been completed

- [x] Unit tests
- [x] Component tests

## Developer Checklist

- [x] I have performed a self-review of my own code
- [x] I have run the [code formatting checks](../README.md#code-quality)
- [x] I have run the [code quality checks](../README.md#code-quality)
- [x] New code meets [standards](https://nhsd-confluence.digital.nhs.uk/display/DI/DI+Ways+of+Working) agreed by the team
- [x] Unit test code coverage is at or above 80%
- [x] New and existing unit tests pass locally with my changes
- [x] Tests have added that prove my fix is effective or that my feature works (Component tests)
- [x] I have made corresponding changes to the documentation
- [x] I have cleaned down my environment (if created)

## Code Reviewer Checklist

- [ ] I have run the unit tests and they run correctly
- [ ] I can confirm the changes have been tested or approved by a tester
- [x] I can confirm no remaining infrastructure is left over from this branch

### CODE REVIEW

**Review by mabe13 — APPROVED** [review]
LGTM 👍

File: build/automation/var/project.mk [hunk 1]
```diff
@@ -24,8 +24,8 @@ DEPLOYMENT_SECRETS = $(PROJECT_ID)-$(PROFILE)/deployment
 AWS_SAME_ACCOUNT_DOCKER_REGISTRY = $(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_DEFAULT_REGION).amazonaws.com/$(PROJECT_GROUP_SHORT)/$(PROJECT_NAME_SHORT)
 PYTHON_VERSION = $(PYTHON_VERSION_MAJOR).$(PYTHON_VERSION_MINOR)
```
Comment (JackPlowman) [inline build/automation/var/project.mk:25]: If I understand the issue correctly then this variable must be removed.

**Review by JackPlowman — CHANGES_REQUESTED** [review]
One small change required. Please correct me if I am incorrect in my understanding of the bug.

**Comment by JackPlowman** [comment]
@ed-sparkes Before you merge the code please can you undeploy your branch using `make undeploy PROFILE=task`

File: build/automation/var/project.mk (hunk 1 above)
Comment (ed-sparkes) [reply build/automation/var/project.mk:25]: You are correct, not sure how that crept back in, removing now ...

**Review by JackPlowman — APPROVED** [review]
LGTM 👍

**Review by mabe13 — APPROVED** [review]

**Pull request merged**